src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -227,6 +227,10 @@
     <value>The {0}'s timestamp certificate has an unsupported signature algorithm.</value>
     <comment>0 - Signature friendly name</comment>
   </data>
+  <data name="VerifyError_TimestampUnsupportedDigestAlgorithm" xml:space="preserve">
+    <value>The {0}'s timestamp signature has an unsupported digest algorithm.</value>
```

Comment:
Should this be enhanced by calling out the name of the digest algorithm? 

@dtivel

Reply:
I think the OID is sufficient.

Reply:
There are lots of places where we could do this, so I'm OK with making this change here and everywhere in a separate PR.  But open a separate issue for it.

Reply:
Created an issue for that https://github.com/NuGet/Home/issues/7926